emscripten/npm/src/emscripten-proxy.js

```diff
@@ -102,6 +102,9 @@ function getToolkitFunction(VerovioModule, method) {
     // char *renderData(Toolkit *ic, const char *data, const char *options)
     mapping.renderData = VerovioModule.cwrap("vrvToolkit_renderData", "string", ["number", "string", "string"]);
 
+    // char *renderToExpansionMap(Toolkit *ic)
```

Comment:
We always have  a `["number"]` parameter even for function without any. The number is the pointer to the toolkit. See `getVersion()`